### PR TITLE
Virtual aliases

### PR DESCRIPTION
**Pull request by mugwhump**
Adds option to use virtual aliases for non-local email forwarding. Added example to documentation.

I also added $mydomain to mydestination in the config file, since $mydomain is often different from $myhostname.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +48,20 @@ A simple example that doesn't use SASL relaying:
...
+---
+- hosts: all
+  roles:
+    - postfix
+  vars:
+    - postfix_virtual_aliases:
+      - virtual: webmaster@yourdomain.com
+        alias: personal_email@gmail.com
+      - virtual: billandbob@yourdomain.com
+        alias: bill@gmail.com, bob@gmail.com
```
Comment (tersmitten) [inline README.md]: Have you considered to make this a list?

**Comment by tersmitten** [comment]
Thanks for your contribution, I would love to add this feature

File: README.md (hunk 1 above)
Comment (mugwhump) [inline README.md]: Do you mean the line where it says
`alias: bill@gmail.com, bob@gmail.com`
No, you can just write them on one line separated by commas and postfix is fine with it.

**Comment by tersmitten** [comment]
> I also added $mydomain to mydestination in the config file, since $mydomain is often different from $myhostname.

Is this needed for `virtual_aliases` to work. The added `$mydomain` causes problems in our setup :-(

**Comment by tersmitten** [comment]
@mugwhump Make sure to have a look at https://github.com/Oefenweb/ansible-postfix/releases/tag/v2.0.0, this might break your setup (but is easy to fix :-)).

**Pull request merged**